windows/hid.c

```diff
@@ -367,6 +389,7 @@ int HID_API_EXPORT hid_init(void)
 			register_global_winapi_error(L"resolve DLL functions");
 			return -1;
 		}
+		hid_hotplug_context.mutex = CreateMutexA(NULL, FALSE, NULL);
```

Comment:
it shouldn't be done under `HIDAPI_USE_DDK`
we probably need to move only the lookup functions under the `HIDAPI_USE_DDK` and keep the `hidapi_initialized` flag and `hid_hotplug_context` initialisation

also, `hid_exit()` should deinitialize `hid_hotplug_context` as well

Reply:
Didn't notice the macro check, should be fixed now.